### PR TITLE
Pylint 2.5.0 fixes

### PR DESCRIPTION
**Pull request by VladimirSlavik**
This consists of fixes to anaconda codebase that I couldn't match to pylint bugs.

### CODE REVIEW

File: pyanaconda/modules/storage/bootloader/base.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ class BootLoaderError(Exception):
     pass
 
 
-class Arguments(OrderedSet):
+class Arguments(OrderedSet):  # pylint: disable=duplicate-bases
```
Comment (jkonecny12) [inline pyanaconda/modules/storage/bootloader/base.py:91]: This seems to me like a pylint error. Or do you know why it is happening?

**Comment by VladimirSlavik** [comment]
@jkonecny Good point about the gui tests. Not sure if we want to spend time on them now, though? I will file a task for later anyway.

File: pyanaconda/modules/storage/bootloader/base.py (hunk 1 above)
Comment (VladimirSlavik) [reply pyanaconda/modules/storage/bootloader/base.py:91]: Yes, I think it's a pylint error. `Arguments` inherits from a standard library class, and as far as I can reach, there is no sign of multiple parents.

Do you think a FIXME would be good here?

**Comment by jkonecny12** [comment]
Thanks @VladimirSlavik !

**Comment by VladimirSlavik** [comment]
Jenkins, test this please.

File: pyanaconda/modules/storage/bootloader/base.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/modules/storage/bootloader/base.py:91]: Btw. OrderedSet is not a standard library class :-)

Comment (VladimirSlavik) [reply pyanaconda/modules/storage/bootloader/base.py:91]: Mea culpa...

Comment (VladimirSlavik) [reply pyanaconda/modules/storage/bootloader/base.py:91]: Added the FIXME for this one.

**Comment by VladimirSlavik** [comment]
Update: added a fixme note for the duplicate bases pragma.

**Comment by VladimirSlavik** [comment]
Jenkins, test this please.

**Pull request merged**